Swiftfin tvOS/Views/SettingsView/VideoPlayerSettingsView.swift

```diff
@@ -57,6 +62,12 @@ struct VideoPlayerSettingsView: View {
                 } footer: {
                     Text("Settings only affect some subtitle types")
                 }
+
+                Section {
+
+                    Toggle("Pause on background", isOn: $pauseOnBackground)
```

Comment:
I'm fine with the way the settings are for now but I may clean them up in the future to something different.